tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "allowUnusedLabels": false,
     "exactOptionalPropertyTypes": true,
     "noImplicitOverride": true,
-    "noPropertyAccessFromIndexSignature": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
Why did you turn off this rule?

Reply:
Cause when using the styles module you have to use an inconvenient systemaxis `classname={style['someStyle']}` instead of `classname={style.someStyle}`